README.md

```diff
@@ -19,7 +19,7 @@ Welcome to My Capstone
 
 ## Live Demo
 
-- [Live demo](https://joseheco.github.io/Capstone1/)
+- [Live demo](https://joseheco.github.io/CapStone1/index.html)
 
 ## Prerequisites
```

Comment:
- [ ] Let's update your `MIT` file. Right now it has placeholders about the LICENSE. These should be filled with the type of app you made and your name.